packages/video_player/video_player_avfoundation/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -264,9 +268,20 @@ - (instancetype)initWithPlayerItem:(AVPlayerItem *)item
   // video streams (not just iOS 16).  (https://github.com/flutter/flutter/issues/109116). An
   // invisible AVPlayerLayer is used to overwrite the protection of pixel buffers in those streams
   // for issue #1, and restore the correct width and height for issue #2.
+  // It is also used to start picture-in-picture
   _playerLayer = [AVPlayerLayer playerLayerWithPlayer:_player];
+  // By default picture-in-picture shows a placeholder where the original video was playing.
```

Comment:
No that is not possible. I tried this. But that just results in the PiP not working at all. 

Opacity = 0 => not working
Opactity > 0 => working

Reply:
That doesn't answer my question; you're still describing opacity there. The comment you wrote says "By default picture-in-picture shows a placeholder ...". Saying that something happens "by default" suggests that there is a way to change that behavior.

So if there is a way to tell the system **not** to show a placeholder, then why isn't the PR doing that, instead of showing the placeholder but hacking around it with opacity?

Reply:
There is no way to say "do not show the placeholder". How should I phrase it?